content/terms_and_definitions/terms_and_definitions.adoc

```diff
@@ -6,7 +6,276 @@ ifndef::include-only-once[]
 include::{root-path}_config.adoc[]
 endif::[]
 
-TODO: Add terms and definitions
+3D Application Terms::
+(To be defined)
 
-Static road network::
-Collection of connected roads enriched by static objects that do not change during runtime of a simulation.
+* Source: 
+* Notes: Taken over from terminology collection list on Sharepoint.
+
+3D Model::
+A (mathematical) geometrical, coordinate-based representation of the surface of an object.
+
+* Source: 
+* Notes: 
+
+3D Environment::
+(To be defined)
+
+* Source: 
+* Notes: 
+
+Asset::
+A 3D geometry combined with assigned material data and specific meta data.
+
+* Source:
+* Notes: 
+
+Camera sensor::
+(To be defined)
+
+* Source: 
+* Notes: 
+
+Closed loop::
+(To be defined)
+
+* Source: 
+* Notes: 
+
+Co-Simulation::
+Simulating multiple independent models connected with interfaces.
+
+* Source: 
+* Notes: 
+
+Environment::
+(To be defined)
+
+* Source: 
+* Notes: Differentiate between 3D environment and simulation environment.
+
+Ground truth::
+True static and dynamic states of all objects as defined by content creator or observed by omniscient observer (e.g. simulation platform).
+
+* Source: 
+* Notes: 
+
+Label::
+(To be defined)
+
+* Source: 
+* Notes: There is no definition for "Label" in OpenLABEL.
+
+Lidar sensor::
+(To be defined)
+
+* Source:
+* Notes:
+
+Material assignment::
+A mapping of material names to parts of 3D models.
+
+* Source: 
+* Notes:
+
+Material classification::
+Sorting materials into classes.
+
+* Source: 
+* Notes:
+
+Material property::
+Intensive physical or chemical properties of a material 
+
+* Source: 
+* Notes: Open Point: What about geometric properties, i.e. roughness, thickness?
+
+Material::
+(To be defined)
+
+* Source:
+* Notes:
+
+Measurement::
+(To be defined)
+
+* Source: 
+* Notes: We would take different meanings compared to an automotive testing domain.
+
+Model::
+(To be defined)
+
+* Source: 
+* Notes: Differentiate between 3D model and other models, e.g. sensor models, logics models, vehicle models.
+
+Multispectral/hyperspectral::
+Wide range of wavelengths.
+
+* Source: 
+* Notes:
+
+Normals (face, vertex normals)::
+(To be defined)
+
+* Source: 
+* Notes: Look up normal vector on surface.
+
+Object::
+An object ID can be assigned, discrete meshes, defined 3D boundaries.
+
+* Source: 
+* Notes:
+
+Object Part / Subtype::
+(To be defined)
+
+* Source: 
+* Notes: What is an object and what is an object part?
+
+Object Type::
+Class of objects with common properties
+
+* Source: 
+* Notes:
+
+Open loop::
+(To be defined)
+
+* Source: 
+* Notes:
+
+Opacity | Translucence | Transparency::
+(To be defined)
+
+* Source: 
+* Notes: Taken over from terminology collection list on Sharepoint.
+
+Parameter File::
```

Comment:
May change to Material Parameter File? Then it would be a file in json format containing defined material parameters according to the standard.